content/articles/2021sy/blog/jisyukenkyu.md

```diff
@@ -0,0 +1,17 @@
+Title: 自主研究奨励事業2020
+Date: 2021.05.06
+Modified: 2021.05.06
+Tags: Machine Learning
+Authors: 安部,梅田,富田
+
+学部学生による自主研究奨励事業、全学選抜自主研究成果発表会
+
+2021/04/30に銀杏会館でとりおこなわれた発表会にて優秀賞をいただきました。
+![写真]({attach}./images/jisyukenkyu_figs/8519.jpg)
+テーマは「眼底画像から緑内障か否か判断する深層学習モデル開発」でした。
+アドバイザー教員の三木 篤也先生をはじめ、様々な方にご支援いただきましたことを心から感謝申し上げます。
+計算機として、[医学科学生用計算機]({filename}/pages/student_server.md)を活用しました。
+
+詳細は[こちら](https://ir.library.osaka-u.ac.jp/repo/ouka/all/search/200220010496/?lang=0&cate_schema=3000&mode=0&cflg=1&codeno=journal)となります。
```

Comment:
The link should be `http://hdl.handle.net/11094/80643`.

Reply:
Can you use this image in your paper?
![image](https://user-images.githubusercontent.com/41336057/117382256-14075480-af19-11eb-9a8c-f72e67017b64.png)

Reply:
OK.